HotelBooking/HotelBooking.BusinessLogic/Services/Implementation/ReservationService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using HotelBooking.BusinessLogic.Dtos;
+using HotelBooking.BusinessLogic.Services.Abstraction;
+using HotelBooking.DataAccessLayer.Repositories.Interfaces;
+
+namespace HotelBooking.BusinessLogic.Services.Implementation;
+public class ReservationService : IReservationService
+{
+    private IReservationRepository _iReservationRepository;
+
+    public ReservationService(IReservationRepository iReservationRepository)
+    {
+        _iReservationRepository = iReservationRepository;
+    }
+
+    public async Task<ReservationDto> GetReservation(int id)
+    {
+        var reservation = await _iReservationRepository.Get(id);
```

Comment:
Dalej będziemy mieć problem z tym, że reservstion.Rooms i reservation.Guest będą nullami. Chciałbym żebyśmy razem to naprawili, więc nie zmieniaj tutaj w tym aspekcie nic. Jeśli poprawisz pozostałe zmiany, to wdzwonimy się na taki programistyczny live :D i chciałbym Ci pokazać pare rzeczy po prostu i uruchomić razem aplikację, podebugować itd